.env.development

```diff
@@ -14,3 +14,4 @@ NEXT_PUBLIC_DISCORD_GUILD_ID = '365559264850477066'
 NEXT_PUBLIC_AUTH_COOKIE_DOMAIN=.egghead.af
 NEXT_PUBLIC_AHOY_COOKIE_DOMAIN=next.egghead.af
 NEXT_PUBLIC_ALGOLIA_INDEX_NAME=content_production
+NODE_OPTIONS="--max-old-space-size=16384" vercel dev
```

Comment:
does this do anything? I don't think it will and you'd need to alias it in your system config `.zshrc` or `.bashrc`

`16384` is also super high for most systems. `4048` would be enough, which is still 4GB